heron/api/src/java/org/apache/heron/streamlet/Streamlet.java

```diff
@@ -22,6 +22,8 @@
 import java.util.List;
 
 import org.apache.heron.classification.InterfaceStability;
```

Comment:
I think we may also need to add `CustomSource` to allow users to provide specific spout as the input?

Reply:
Yeah. It is in another PR already. https://github.com/apache/incubator-heron/pull/3032 (may need refactor though)